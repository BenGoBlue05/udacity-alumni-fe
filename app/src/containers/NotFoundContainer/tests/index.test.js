import NotFound from '../index';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';

describe('<NotFound />', () => {
  it('should render with default props', () => {
    const wrapper = shallow(
    	<NotFound />
    );
    expect(wrapper.length).toBe(1);
    expect(wrapper.find('Anchor').length).toBe(1);
    expect(wrapper.find('Paragraph').length).toBe(2);
  });
});
