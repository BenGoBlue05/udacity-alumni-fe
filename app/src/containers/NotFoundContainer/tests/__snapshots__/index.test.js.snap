exports[`<NotFound /> should render with default props 1`] = `
<div>
  <Box
    announce={false}
    direction="column"
    focusable={true}
    pad="large"
    responsive={true}
    tag="div"
    textAlign="center">
    <Box
      align="center"
      announce={false}
      direction="column"
      focusable={true}
      pad="none"
      responsive={true}
      tag="div">
      <Heading
        tag="h3">
        Hello curious adventurer! : )
      </Heading>
      <Paragraph
        textAlign="center">
        You\'ve wandered into uncharted territory. 404-Land.
        <br />
        If that is right were you belong, then get ready to map!
      </Paragraph>
    </Box>
    <Box
      align="center"
      announce={false}
      direction="column"
      focusable={true}
      pad="none"
      responsive={true}
      tag="div">
      <Heading
        alignContent="center"
        className="problem"
        tag="h1">
        No need for dusty slide-rules
      </Heading>
      <Paragraph
        className="solution">
        Learn to move the numbers on your computer!
      </Paragraph>
      <Anchor
        className="solution-link"
        href="https://www.udacity.com/course/data-analyst-nanodegree--nd002"
        tag="a">
        Data Analyst Nanodegree
      </Anchor>
    </Box>
  </Box>
</div>
`;
